com.unity.shadergraph/Documentation~/TableOfContents.md

```diff
@@ -87,6 +87,10 @@
             * [Blackbody](Blackbody-Node)
             * [Gradient](Gradient-Node)  
             * [Sample Gradient](Sample-Gradient-Node)
+        * High Definition Render Pipeline
+            * [Diffusion Profile](Diffusion-Profile-Node.md)
+            * [Exposure](Exposure-Node.md)
+            * [HD Scene Color](HD-Scene-Color-Node.md)
```

Comment:
i'll need to double check this. we format our node library table of contents based on how they appear inside of the create node menu in the graph, and I can't remember if we actually have multiple "HDRP" folders nestled in various places.

Reply:
I made sure to organise the toc following how it looks in the create node menu, but I was using 2019.4 so it may have changed later on.